source/common/common/logger.h

```diff
@@ -153,9 +154,19 @@ template <Id id> class Loggable {
 #define ENVOY_LOG_TO_LOGGER(LOGGER, LEVEL, ...) ENVOY_LOG_##LEVEL##_TO_LOGGER(LOGGER, ##__VA_ARGS__)
 
 /**
+ * Convenience macro to get logger.
+ */
+#define ENVOY_LOGGER() __log_do_not_use_read_comment()
+
+/**
+ * Convenience
```

Comment:
nit: "Convenience macro to flush logger."

Reply:
_facepalm_. Done